examples/README.md

```diff
@@ -0,0 +1,27 @@
+# Fishjam Room Manager
+
+## Running (in dev mode)
+All available options are defined in [arguments.py](room_manager/arguments.py).
+
+```sh
```

Comment:
**nitpick**
So far in most places I think we are using `console` for commands markdown. However idk if there is any difference at all
```suggestion
```console
```